controllers/networkchaos/partition/types.go

```diff
@@ -196,6 +197,10 @@ func (r *Reconciler) Apply(ctx context.Context, req ctrl.Request, chaos v1alpha1
 
 	err = m.Commit(ctx)
 	if err != nil {
+		// if pod is not found or not ready, wait next time.
+		if err == podnetworkmanager.ErrPodNotFound || err == podnetworkmanager.ErrPodNotRunning {
+			return err
+		}
```

Comment:
why need these code, seems that  it will return err whatever the error type is

Reply:
The other errors would do `r.Log.Error(err, "fail to commit")`, I don't think we need the full-stack debug log if the pod is deleted or not running.

Reply:
```
if err != podnetworkmanager.ErrPodNotFound && err != podnetworkmanager.ErrPodNotRunning {
    r.Log.Error(err, "fail to commit")
}
return err
```

maybe this is better 😅

Reply:
Okay.